ember-cli-build.js

```diff
@@ -26,6 +26,8 @@ module.exports = function (defaults) {
   })
 
   app.import('bower_components/sinonjs/sinon.js')
+  // client-side template compilation for the abstract-input demo page
+  app.import('bower_components/ember/ember-template-compiler.js')
```

Comment:
Was this necessary? Could you not put a custom component in `tests/dummy/app/components/` instead?